server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfilesWs.java

```diff
@@ -26,13 +26,17 @@
   public static final String API_ENDPOINT = "api/qualityprofiles";
 
   private final QProfileRestoreBuiltInAction resetAction;
+  private final QProfileSearchAction searchAction;
   private final RuleActivationActions ruleActivationActions;
   private final BulkRuleActivationActions bulkRuleActivationActions;
 
+
   public QProfilesWs(QProfileRestoreBuiltInAction resetAction,
+                     QProfileSearchAction searchAction,
```

Comment:
It could be interesting here only inject only some BaseQProfileWsAction classes (see for instance IssuesWs and BaseIssuesWsAction). 
It allows to more easily create this class in tests